torch/__init__.py

```diff
@@ -343,6 +343,7 @@ def _load_global_deps() -> None:
             "cufft": "libcufft.so.*[0-9]",
             "curand": "libcurand.so.*[0-9]",
             "nvjitlink": "libnvJitLink.so.*[0-9]",
+            "cufile": "libcufile.so.*[0-9]",
```

Comment:
Thank you for fixing this!!

I don't fully understand how this bit of code works, but since we only use cufile in cuda 12.6 and 12.8 builds, do we need an if statement for that here?

Otherwise, perhaps this would break the other binaries in a similar way that happened for 2.5.0 :(